spec/models/ingredients_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe Ingredient, type: :model do
+  describe "relationships" do
+    it { should have_many :ingredient_dishes }
+    it { should have_many(:dishes).through(:ingredient_dishes) }
+  end
+end
```

Comment:
Relationship tests look good, and great work testing the instance methods at the unit and end-to-end (feature) level